### ISSUE TITLE
Custom"look at" method for rigidbody needs updating

### ISSUE THREAD

**Issue by btzr-io** [godotengine/godot-docs#3835]
**Your Godot version:**
3.2.2

**Issue description:**
As the docs mention a `"custom look_at() method that will work reliably with rigid bodies"` is provided as an example function, however this don't really work as expected and only is able to follow target on a 180 degrees on the x axis.

See: https://stackoverflow.com/questions/59146365/how-to-make-a-rigidbody-look-at-another-spatial

There is no warning or observation about this on the docs. An updated function or explanation on how to make it work as expected ( full rotation from any direction ) will be nice to have. 

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/stable/tutorials/physics/rigid_body.html#the-look-at-method

**Comment by AlecTroemel** [comment]
I noticed this the other day. I fumbled into this method which "works".

```gdscript
func look_follow(state, current_transform, target_position):
	var up_dir = Vector3(0.0, 1.0, 0.0)
	var cur_dir = current_transform.basis.xform(Vector3(0.0, 0.0, 1.0))

        // use built in vector methods
	var target_dir = current_transform.origin.direction_to(target_position)
	var rotation_angle = -cur_dir.angle_to(target_dir) / 10

        # use “perp dot product” to get correct sign
	if (cur_dir.x * target_dir.z) - (cur_dir.z * target_dir.x) < 0:
		rotation_angle = -rotation_angle;

	state.set_angular_velocity(up_dir * (rotation_angle / state.get_step()))
```

however I am not sure why I needed to start with a negative rotation angle, then divide it by 10 🤔 
hopefully someone with more powerful vector math skills can clean this up so we can update the docs!

**Comment by stevecotton** [comment]
I'm looking at the same example's use of `up_dir`, and wondering if the "will work reliably with rigid bodies:" should have "assuming that you're making a game where the player character's Y axis is locked to point in the same direction as the world's Y axis" added to it. Testing on Godot4, the value passed to `set_angular_velocity` appears to be interpreted using the world's axes rather than local axes.

**Comment by skyace65** [comment]
Fixed by #8179
